.clang-format

```diff
@@ -0,0 +1,35 @@
+---
+BasedOnStyle: Google
+---
+Language: Cpp
+ColumnLimit: 78
```

Comment:
I have always hated this limitation. I think 98 is a much better value. But of course, YMMV.

Reply:
I think 98 is acceptable too. Tbh I'm not too annoyed with the 78 limit, but the reason why I prefer it is mainly because it's what the rest of the codebase  uses (for the most part), and also seems most "standard".